bandit-github/bandit_github/formatter.py

```diff
@@ -1,12 +1,17 @@
 import itertools
 import json
 import os
+from typing import IO
 
 import requests
 from bandit.core import constants, docs_utils, test_properties
 
+BANDIT_COMMENT_ON_PULL_REQUEST = os.environ.get(
+    "BANDIT_COMMENT_ON_PULL_REQUEST", "False"
+)
```

Comment:
```suggestion
BANDIT_COMMENT_ON_PULL_REQUEST = os.environ.get("BANDIT_COMMENT_ON_PULL_REQUEST")
```

`BANDIT_COMMENT_ON_PULL_REQUEST` will always be truthy if it is set. Even if it is not set, because:

```python
>>> foo = "False"
>>> if foo:
...     print("oops")
... 
oops
```

Reply:
Ah I didn't know that `os.environ.get` will return `None` by default, very nice 👍🏼